build/gen_stub.php

```diff
@@ -1262,29 +1262,89 @@ function initPhpParser() {
 }
 
 $optind = null;
-$options = getopt("fh", ["force-regeneration", "parameter-stats", "help"], $optind);
+$options = getopt("fh", ["force-regeneration", "parameter-stats", "help", "verify"], $optind);
```

Comment:
I used a general name for the verification in case we want to extend the validation to other things.